src/core/utils/index.js

```diff
@@ -366,3 +367,11 @@ export function addonHasVersionHistory(addon) {
     ADDON_TYPE_THEME,
   ].includes(addon.type);
 }
+
+export const getRelativePath = (filename, dirname = __dirname) => {
+  if (dirname) {
+    return path.relative(path.resolve(dirname, '../../../'), filename);
```

Comment:
This looks like it won't work on Windows and we might forget to update it in case `utils/index.js` gets moved.

Since it only needs to guarantee a consistent path, what about a simpler approach more like this?

```js
let fileId = filename;
if (!fileId.startsWith('src')) {
  fileId = fileId.replace(/^.*src/, 'src');
}
```

Reply:
sounds good, will try this.

Reply:
Note: I assumed Node would fix the Windows vs UNIX difference as done in many languages.